lib/helpers/calculator_helpers.js

```diff
@@ -495,4 +481,84 @@ module.exports = class CalculatorHelpers {
     };
     return elmFunction;
   }
+
+  static map;
+  static systemUriByName(systemName) {
+    if(this.map === undefined) {
```

Comment:
Is this an acceptable pattern for establishing a cache? 

Comparing the static property against `undefined` worked in unit tests, but I'm not sure if there any "gotchas" with it.